### DISCUSSION TITLE
Chào mừng anh em đến với nhóm 5!

### DISCUSSION THREAD

**Discussion by kyozun** [kyozun/Group5-ecommerce-food#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kyozun** [comment]
hello anh em

**Comment by nguyentrongbut** [comment]
+1

**Comment by nguyenluu1203** [comment]
+2

**Comment by nguyentrongbut** [comment]
-1

**Comment by nguyentrongbut** [comment]
💯
